script.js

```diff
@@ -0,0 +1,46 @@
+let books = [];
+
+const addBtn = document.getElementById('addBook');
+const listBox = document.getElementById('list');
+
+function displaybooks() {
+  const book = localStorage.getItem('book');
+  if (book === null) {
+    books = [];
+  } else {
+    books = JSON.parse(book);
+  }
+  let bookscontainer = '';
+  books.forEach((item, index) => {
+    bookscontainer += `<h3 id="">${item.title}</h3>
+            <p id="">${item.auther}</p>
+            <button type="button" onclick="Remove(${index})">Remove</button>`;
+  });
+  listBox.innerHTML = bookscontainer;
+}
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const title = document.getElementById('bookName').value;
+  const auther = document.getElementById('author').value;
+  const book = localStorage.getItem('book');
+  if (book === null) {
+    books = [];
+  } else {
+    books = JSON.parse(book);
+  }
+  books.push({
+    title,
+    auther,
+  });
+  localStorage.setItem('book', JSON.stringify(books));
+  displaybooks();
+});
+
+function Remove(index) { // eslint-disable-line no-unused-vars
+  const book = localStorage.getItem('book');
+  books = JSON.parse(book);
+  books.splice(index, 1);
+  localStorage.setItem('book', JSON.stringify(books));
+  displaybooks();
+}
```

Comment:
- [ ] You did a fantastic job storing all the books using LocalStorage :+1:, but I have noticed that when the user refreshes the page the content is not loading. To make it render you can call `displaybooks();` function at the end of your JavaScript file :+1: 

**Your project**

![Screenshot from 2023-02-21 02-53-20](https://user-images.githubusercontent.com/105117832/220284695-2ba5e105-6b22-4c55-9e6e-b9b07ee3c7e8.png)